scripts/west_commands/runners/ezflashcli.py

```diff
@@ -3,43 +3,49 @@
 # SPDX-License-Identifier: Apache-2.0
 
 '''Runner for flashing with ezFlashCLI.'''
+import shlex
+
 from runners.core import ZephyrBinaryRunner, RunnerCaps
 
 DEFAULT_EZFLASHCLI = "ezFlashCLI"
 
 class EzFlashCliBinaryRunner(ZephyrBinaryRunner):
     '''Runner front-end for ezFlashCLI'''
 
-    def __init__(self, cfg, tool, sn, erase=False, reset=True):
+    def __init__(self, cfg, tool, tool_opt=[], erase=False, reset=True):
         super().__init__(cfg)
         self.bin_ = cfg.bin_file
 
         self.tool = tool
-        self.sn_arg = ['-j', f'{sn}'] if sn is not None else []
         self.erase = bool(erase)
         self.reset = bool(reset)
 
+        self.tool_opt = []
+        for opts in [shlex.split(opt) for opt in tool_opt]:
+            self.tool_opt += opts
+
     @classmethod
     def name(cls):
         return 'ezflashcli'
 
     @classmethod
     def capabilities(cls):
-        return RunnerCaps(commands={'flash'}, erase=True, reset=True)
+        return RunnerCaps(commands={'flash'}, tool_opt=True, erase=True, reset=True)
+
+    @classmethod
+    def tool_opt_help(cls) -> str:
+        return "Additional options for ezFlashCLI e.g. '-j <serial_number>'"
```

Comment:
This doesn't look right, there is a dedicated device ID in `scripts/west_commands/runners/core.py`:
```
    - dev_id: whether the runner supports device identifiers, in the form of an
      -i, --dev-id option. This is useful when the user has multiple debuggers
      connected to a single computer, in order to select which one will be used
      with the command provided.
```
This should be using that

Reply:
Added dev_id capability.